src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: $font-weight-medium;
+    font-size: 12px;
+    line-height: 18px;
```

Comment:
try to avoid line-height, if you want to align content use display: flex for parent tag + align-items: center or, with justify-content: center

Reply:
In this case, line-height allows to keep the required distance between the 2 lines of title text.
If I remove it, the whole arrangement of elements breaks (